### ISSUE TITLE
BSD sed(1) y command + backslash portability

### ISSUE THREAD

**Issue by ikazont** [ikazont/netbsd#130]
**[Original report](https://bitbucket.org/tnozaki/netbsd/issues/130) by Takehiko NOZAKI (Bitbucket: [tnozaki](https://bitbucket.org/tnozaki), GitHub: [ikazont](https://github.com/ikazont)).**

----------------------------------------

the spec says:
```
http://pubs.opengroup.org/onlinepubs/9699919799/utilities/sed.html

If a <backslash> character is immediately followed by a <backslash> character
in string1 or string2, the two <backslash> characters shall be counted as a
single literal <backslash> character.
```

FreeBSD + OpenBSD is already fixed.
https://svnweb.freebsd.org/base?view=revision&revision=197361
http://cvsweb.openbsd.org/cgi-bin/cvsweb/src/usr.bin/sed/compile.c#rev1.33

TNF HEAD is ok(with uncertain merge commit...)
http://cvsweb.netbsd.org/bsdweb.cgi/src/usr.bin/sed/compile.c#rev1.41

**Comment by ikazont** [comment]
**Original comment by Takehiko NOZAKI (Bitbucket: [tnozaki](https://bitbucket.org/tnozaki), GitHub: [ikazont](https://github.com/ikazont)).**

----------------------------------------

try following:
```
#!shell
echo '.\\home' | sed 'y|\\|/|'
```

**Comment by ikazont** [comment]
**Original comment by Takehiko NOZAKI (Bitbucket: [tnozaki](https://bitbucket.org/tnozaki), GitHub: [ikazont](https://github.com/ikazont)).**

----------------------------------------

fixed <<cset:64aac0c>>

**Comment by ikazont** [comment]
**Original comment by Takehiko NOZAKI (Bitbucket: [tnozaki](https://bitbucket.org/tnozaki), GitHub: [ikazont](https://github.com/ikazont)).**

----------------------------------------

BUGFIX: Issue #130 - stupid regression introduced by <<cset:ca5fac1>>
